src/app/frontend/replicasetdetail/replicasetdetail.html

```diff
@@ -150,29 +280,112 @@
         <table class="kd-replicasetdetail-table" cellspacing="0" cellpadding="15">
           <thead>
           <tr>
-            <th class="kd-replicasetdetail-table-cell">
-              Message<i class="material-icons kd-replicasetdetail-table-icon">help</i>
+            <th class="kd-replicasetdetail-table-header">
+              <a class="kd-replicasetdetail-pointer" ng-click="ctrl.sortEvents('message')">
+                <i ng-show="ctrl.areEventsSortedBy('message', true)"
+                   class="material-icons kd-replicasetdetail-sort-icon">
+                  arrow_downward
+                </i>
+                <i ng-show="ctrl.areEventsSortedBy('message', false)"
+                   class="material-icons kd-replicasetdetail-sort-icon">
+                  arrow_upward
+                </i>
+                Message
+              </a>
+              <i class="material-icons kd-replicasetdetail-help-icon">
+                help
+                <md-tooltip>Event message.</md-tooltip>
+              </i>
```

Comment:
Let's try to avoid so much boilerplate code. We can use directive here and just tell to sort by `message` `name`, etc.